cmds/tui.c

```diff
@@ -1662,7 +1662,7 @@ static void win_header_info(struct tui_window *win,
 static void win_footer_info(struct tui_window *win,
 			    struct uftrace_data *handle)
 {
-	char buf[256];
+	char buf[COLS * LINES + 1];
```

Comment:
I think the problem is not the buf is small, 256 is big enough to save the version string.  The real problem is the size we passed to snprintf(), it should be the sizeof(buf) not the COLS.